### ISSUE TITLE
Property 'default' added to stubs when proxying using rewiremock/node 

### ISSUE THREAD

**Issue by atorma** [theKashey/rewiremock#22]
I'm doing something like this in a Node.js Mocha test

```
const rewiremock = require('rewiremock/node').default
const myService = rewiremock.proxy('path/to/service', {
   'path/to/data': dataStub
});
```

where `dataStub` is an data object. `myService` goes through each key of `dataStub`. Now, rewiremock adds an extra property `default` to the stubs. This interferes with what the service is doing.

Is there a way to prevent rewiremock from adding that extra `default` property to a stub? Should it do this at all in a Node.js environment?

**Comment by theKashey** [comment]
That's my bad. It is a logic to _partially_ mimic webpack behaviour, and that is needed only for dynamic imports.
There is no way to disable it now, but I will make it possible asap.

**Comment by atorma** [comment]
Excellent, thanks!

**Comment by theKashey** [comment]
Should be fixed in 3.7.2.
Not actually "fixed" - default will still exists, but will be not enumerated.
